config/settings/base.py

```diff
@@ -361,12 +367,19 @@
         CONNECTID_CLIENT_SECRET,
     ),
 }
+OAUTH2_PROVIDER_APPLICATION_MODEL = "oauth2_provider.Application"
+
 
+# Twilio settings
 TWILIO_ACCOUNT_SID = env("TWILIO_SID", default=None)
 TWILIO_AUTH_TOKEN = env("TWILIO_TOKEN", default=None)
 TWILIO_MESSAGING_SERVICE = env("TWILIO_MESSAGING_SERVICE", default=None)
 MAPBOX_TOKEN = env("MAPBOX_TOKEN", default=None)
 
 OPEN_EXCHANGE_RATES_API_ID = env("OPEN_EXCHANGE_RATES_API_ID", default=None)
 
-OAUTH2_PROVIDER_APPLICATION_MODEL = "oauth2_provider.Application"
+# Waffle Settings
+WAFFLE_FLAG_MODEL = "flags.Flag"
```

Comment:
Despite not using flags yet, we need to specify a custom model before first deploy if we will want one later, so this is to keep our options open, but is not to be used.